From 122570401dd50a369f4a26ecfc29f409d61a8de5 Mon Sep 17 00:00:00 2001
From: Stella Laurenzo <stellaraccident@gmail.com>
Date: Fri, 11 Apr 2025 17:34:05 -0700
Subject: [PATCH 8/8] Do not use GTEST_BOTH_LIBRARIES variable.

* This is a legacy variable that is not recommended for modern use and is only available via the GTest find module, not the actual config provided by the package.
* Recommended usage is just to depend on the libraries.
* Also removes GTEST_INCLUDE_DIRS, since that is implicit in depending on the library.
---
 clients/gtest/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/clients/gtest/CMakeLists.txt b/clients/gtest/CMakeLists.txt
index 64cd576a..a57100f2 100644
--- a/clients/gtest/CMakeLists.txt
+++ b/clients/gtest/CMakeLists.txt
@@ -62,10 +62,10 @@ target_include_directories( hipblaslt-test
     $<BUILD_INTERFACE:${HIP_INCLUDE_DIRS}>
     $<BUILD_INTERFACE:${BLAS_INCLUDE_DIR}>
     $<BUILD_INTERFACE:${BLIS_INCLUDE_DIR}> # may be blank if not used
-    $<BUILD_INTERFACE:${GTEST_INCLUDE_DIRS}>
 )
 message("BLIS_INCLUDE_DIR=" ${BLIS_INCLUDE_DIR})
-target_link_libraries( hipblaslt-test PRIVATE ${BLAS_LIBRARY} ${GTEST_BOTH_LIBRARIES} roc::hipblaslt )
+target_link_libraries( hipblaslt-test PRIVATE ${BLAS_LIBRARY} 
+  GTest::gtest GTest::gtest_main roc::hipblaslt )
 
 if( NOT BUILD_CUDA )
   target_link_libraries( hipblaslt-test PRIVATE hip::host hip::device )
-- 
2.43.0

